crates/bevy_pbr/src/light.rs

```diff
@@ -368,44 +368,30 @@ pub struct Clusters {
 }
 
 impl Clusters {
-    fn new(tile_size: UVec2, screen_size: UVec2, z_slices: u32, near: f32, far: f32) -> Self {
+    fn new(screen_size: UVec2, dimensions: UVec3, near: f32, far: f32) -> Self {
+        debug_assert!(screen_size.x > 0 && screen_size.y > 0);
+        debug_assert!(dimensions.x > 0 && dimensions.y > 0 && dimensions.z > 0);
```

Comment:
these asserts should move to update, we are only asserting for the initial/dummy call here.

Reply:
I moved the dimensions assert, but asserting on zero screen size would "break" the fix because we need to run the update logic each frame. Instead, I made it "gracefully" handle screen sizes of zero (which result in zero values) by doing `.max(VecType::ONES)`